File: rename_target.rb
# encoding: utf-8
require "xcodeproj"

project_path = "ios/App/App.xcodeproj"
project = Xcodeproj::Project.open(project_path)

# Get the main target
target = project.targets.first
target.name = "棉花糖夥伴"

# Update PRODUCT_NAME in build configurations
target.build_configurations.each do |config|
  config.build_settings["PRODUCT_NAME"] = "棉花糖夥伴"
end

# Update the project name
project.root_object.name = "棉花糖夥伴"

project.save
puts "Done!"

File: rename_target.js
const xcodeproj = require('xcodeproj');

const projectPath = 'ios/App/App.xcodeproj';
const project = xcodeproj.open(projectPath);

// Get the main target
const target = project.targets[0];
target.name = '棉花糖夥伴';

// Update PRODUCT_NAME in build configurations
target.buildSettings('Debug'). PRODUCT_NAME = '棉花糖夥伴';
target.buildSettings('Release'). PRODUCT_NAME = '棉花糖夥伴';

// Update the project name  
project.rootObject.name = '棉花糖夥伴';

project.save();
console.log('Done!');
